roles/docker/tasks/main.yml

```diff
@@ -28,7 +28,20 @@
     state: started
 
 # Install docker-compose which also installs the docker python library required by some ansible docker modules
-- name: install docker-compose
+- name: install docker
   pip:
-    name: "docker-compose"
+    name: "docker"
     state: present
+
+- name: check ssl_match_hostname
+  stat:
+    path: /usr/lib/python2.7/dist-packages/backports/ssl_match_hostname
+  register: ssl_match_hostname_exists
+  when: ansible_distribution == 'Debian'
+
+- name: copy ssl_match_hostname
+  copy:
+    remote_src: yes
+    src: /usr/local/lib/python2.7/dist-packages/backports/ssl_match_hostname
```

Comment:
Looks like this path has been used multiple times, you can probably store it as a variable.